betterforms/tests.py

```diff
@@ -332,6 +333,7 @@ class Meta:
                 )
         self.TestForm = TestForm
 
+    @unittest.skipIf(django.VERSION < (1, 8), 'HTML output is slightly different in 1.8')
```

Comment:
Yeah. I considered doing that in the first place. I'm not sure why I decided not to in the end, it's clearly the better approach. I'll fix that now!